config/config.go

```diff
@@ -204,6 +216,11 @@ func LoadConfiguration(configPath string) (*Config, error) {
 	}
 	config.configPath = usedConfigPath
 	config.UpdateLastFileModTime()
+
+	if config.LogLevel == "" {
+		config.LogLevel = "INFO"
+	}
```

Comment:
```suggestion
	if config.LogLevel == "" {
		config.LogLevel = logr.LevelInfo
	}
```
once you took care of the previous suggestion, you should be able to just use this directly

Reply:
it's fine to have this anyways, but I also just made a change in the library to make it so if an invalid/unsupported threshold is set, it automatically defaults to INFO